### DISCUSSION TITLE
Not working with new Periodic Notes plugin (beta)

### DISCUSSION THREAD

**Discussion by jarbochov** [liamcain/obsidian-calendar-plugin#242] [General]
Hello,

I'm testing out your new Periodic Notes beta, but when I click on the calendar to create a new daily, it's not applying the template. Not sure if this can be addressed quickly.

**Comment by liamcain** [comment]
Thanks for the report, I've been tracking this <https://github.com/liamcain/obsidian-periodic-notes/issues/100> and plan on fixing it ASAP
